compliance/report.py

```diff
@@ -99,27 +99,86 @@ def get_template_for(self, test_obj, evidence):
             return os.path.join(tmpl_dir, 'default.md.tmpl')
         return tmpl_path
 
-    def _generate_raw_results(self):
-        """Create a check results JSON file stored in the evidence locker."""
-        self.locker.add_content_to_locker(
-            json.dumps(
-                self._munge_chk_results(self.locker.get_reports_metadata()),
-                indent=2,
-                sort_keys=True,
-                separators=(',', ': '),
-                skipkeys=True,
-                default=str
-            ),
-            filename='check_results.json'
-        )
+    def generate_toc(self, rpt_metadata):
+        """
+        Generate a check reports table of contents.
+
+        This method generates a TOC based on all report evidence metadata and
+        appends that TOC to the bottom of an evidence locker's README.md file.
+
+        :param rpt_metadata: Metadata from all report evidence index.json files
+        """
+        readme = 'README.md'
```

Comment:
pathlib might be nice here...

Reply:
how so?

Reply:
I replaced all usage of the `os` package with the `pathlib` package in https://github.com/ComplianceAsCode/auditree-framework/pull/71/commits/5c5a7e607bcb7340b919fd3588f42873e3970aff